src/backend/mod.rs

```diff
@@ -1,11 +1,13 @@
 use event;
 use theme;
 
-// Module is not named `ncurses` to avoir naming conflict
-#[cfg(feature = "ncurses")]
-pub mod curses;
 #[cfg(feature = "termion")]
-pub mod termion;
+mod termion;
+mod curses;
+
+pub use self::curses::*;
+#[cfg(feature = "termion")]
+pub use self::termion::*;
```

Comment:
I'd like to minimize the number of feature-conditional items. For instance, I'm thinking of having something like

```rust
#[cfg(feature = "termion")]
pub use self::termion::TermionBackend as Backend;

#[cfg(feature = "pancurses")]
pub use self::curses::PancursesBackend as Backend;
```
And removing the conditional from `src/lib.rs`.